recipes-robot/fs-overlay/ot3-overlay-mount-points_1.0.bb

```diff
@@ -0,0 +1,20 @@
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM ?= "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
```

Comment:
If this is in #51 let's remove it here

Reply:
removed it from [#51](https://github.com/Opentrons/meta-opentrons/pull/51)